experiments/block-level-dedup/generate_data.py

```
@@ -0,0 +1,156 @@
+            terminal_width = 80
+            try: terminal_width = os.get_terminal_size().columns
+            except OSError: pass
+            print(f"\r{' ' * (terminal_width -1)}\r", end="") 
+
+            elapsed_time = time.time() - start_time
+            gb_written = current_size_bytes / (1024**3)
+            avg_speed_mb_s = 0
+            if elapsed_time > 0 :
+                avg_speed_mb_s = (current_size_bytes / (1024**2)) / elapsed_time
+            
+            print(f"Finished generation!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)